src/catalog/src/durable/impls/persist.rs

```diff
@@ -539,6 +548,10 @@ impl OpenableDurableCatalogState for UnopenedPersistCatalogState {
         }
     }
 
+    fn set_catalog_kind(&mut self, catalog_kind: CatalogKind) {
+        warn!("unable to set catalog kind to {catalog_kind:?}");
+    }
```

Comment:
Uncovered, is this supposed to do something more than warn?

Reply:
No, this is supposed to do nothing but emit a warning. I didn't add a test for this.